bemserver_core/model/sites.py

```diff
@@ -1,11 +1,28 @@
 """Sites"""
+import enum
 import sqlalchemy as sqla
 import sqlalchemy.orm as sqlaorm
 from sqlalchemy.ext.hybrid import hybrid_property
 
 from bemserver_core.database import Base
 from bemserver_core.model import Campaign
 from bemserver_core.authorization import AuthMixin, auth, Relation
+from bemserver_core.exceptions import PropertyTypeInvalidError
+
+
+class PropertyType(enum.Enum):
```

Comment:
Shouldn't this be in a dedicated module? It is not more tied to sites than timeseries.
(If we move it, also move the tests.)

Reply:
Yes, move PropertyType in a new general and dedicated module.